mock/py/mockbuild/package_manager.py

```diff
@@ -671,7 +674,7 @@ def _check_missing(output):
 
 
 class Dnf(_PackageManager):
-    name = 'dnf'
+    name = 'dnf4'
```

Comment:
???
```
 ls -l /usr/bin/dnf
lrwxrwxrwx. 1 root root 5  3. čec 02.00 /usr/bin/dnf -> dnf-3
```
On my F40.
BTW - what is DNF4? DNF3 is what we have now and DNF5 is the rewrite in C++. Right?

Reply:
I was told that `dnf-3` is a leftover from the Python 2 + 3 times; it was normal to add `-3` suffix for Python 3 executable (and it stayed as-is).  But it's a python, which means it is DNF4.

This `name = dnf4` thus IMO correct at this point.